README.md

```diff
@@ -76,6 +76,32 @@ not found
 
 The first layer of the full error output shows a message ("error getting resource 'example-id'") and a single stack frame. The next layer shows the root error ("not found") and the full stack trace.
 
+
+## Formatted error printing
+The default format in `eris` is returned by the method [`NewDefaultFormat()`](https://godoc.org/github.com/morningvera/eris#NewDefaultFormat). Below you can see the what the default formatted error in `eris` might look like.
+
+Errors printed without trace using `fmt.Printf("%v\n", err)`
+```
+even more context: additional context: root error
+```
+
+Errors printed with trace using `fmt.Printf("%+v\n", err)`
+```
+additional context
```

Comment:
this is missing the "even more context" line, probably makes sense to keep them consistent